statsmodels/tsa/vector_ar/tests/test_var_jmulti.py

```diff
@@ -197,42 +193,36 @@ def test_ols_det_terms():
             det_key_ref = "Deterministic term"
             # If there are no det. terms, just make sure we don't compute any:
             if det_key_ref not in results_ref[ds][dt_s]["est"].keys():
-                yield assert_, \
-                      (results_sm[ds][dt_s].coefs_exog.size == 0 and
+                assert (results_sm[ds][dt_s].coefs_exog.size == 0 and
```

Comment:
this should use `assert_` , i.e. numpy instead of pytest rewriting, The test has it's own `err_msg`
(I have no idea what pytest does with the trailing err_msg)